camel/types/enums.py

```diff
@@ -65,7 +65,17 @@ class ModelType(UnifiedModelType, Enum):
     CLAUDE_3_5_SONNET = "claude-3-5-sonnet-20240620"
 
     # Nvidia models
-    NEMOTRON_4_REWARD = "nvidia/nemotron-4-340b-reward"
```

Comment:
we still have [examples/models/nemotron_model_example.py](https://github.com/camel-ai/camel/blob/843acdf4d1b7e2c576fe5814779aedf6db13bc2d/examples/models/nemotron_model_example.py#L18) depending on this enum value. Please update it.

Reply:
Hi Will, thanks for your reviewing. Just fixed this problem and added the model back.

Reply:
The NEMOTRON_4_REWARD is the same as NVIDIA_NEMOTRON_340B_REWARD. Please remove NEMOTRON_4_REWARD and update the [examples/models/nemotron_model_example.py](https://github.com/camel-ai/camel/blob/843acdf4d1b7e2c576fe5814779aedf6db13bc2d/examples/models/nemotron_model_example.py#L18) to use NVIDIA_NEMOTRON_340B_REWARD instead.

Reply:
Hiii Will, thanks for your comment. I just updated the code.